docusaurus/docs/users/howto/self-hosting.md

```diff
@@ -47,11 +61,11 @@ Copy and paste the following `YAML` data into the Cloud Config field. Then repla
 - path: /etc/nixos/host.nix
   permissions: '0644'
   content: |{pkgs, ...}:{   
-    users.users.root.openssh.authorizedKeys.keys = [“# your public key #”];
+    users.users.root.openssh.authorizedKeys.keys = [“/* your public key*/”];
   }
 ```
 
-The `YAML` configuration must also include an instruction to "infect" (overwrite) the contents of your server machine with NixOS. Add the text below to the Cloud Config field. Then compare the infect link below (right after the word "curl") with the link currently visible in the "How do I use it?" section on the [NixOS GitHub page](https://github.com/elitak/nixos-infect). If the link you find on the NixOS GitHub page is different from the one we have here, replace our link with the newer one in your Cloud Config field.
+The `YAML` configuration must also include an instruction to overwrite the contents of your server machine with NixOS. Add the text below to the Cloud Config field. Then compare the link below (right after the word "curl") with the link currently visible in the "How do I use it?" section on the page for this [NixOS installation script](https://github.com/elitak/nixos-infect). If the link you find on that page is different from the one we have here, replace our link with the newer one in your Cloud Config field.
```

Comment:
I would just suggest to go get these instructions from NixOS infect in the first place. And then say "As an example, here's what they currently look like for us.